<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1114</width>
    <height>626</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Recogintion</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>691</width>
      <height>601</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayoutPlot"/>
   </widget>
   <widget class="QSpinBox" name="spinBoxSamples">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>271</width>
      <height>42</height>
     </rect>
    </property>
    <property name="minimum">
     <number>10</number>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
    <property name="singleStep">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>33</y>
      <width>66</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Samples</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>66</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Classes</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxClasses">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>271</width>
      <height>42</height>
     </rect>
    </property>
    <property name="minimum">
     <number>2</number>
    </property>
    <property name="maximum">
     <number>4</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>2</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxFeatures">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>160</y>
      <width>271</width>
      <height>42</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>3</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>66</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Features</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonTrain">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>520</y>
      <width>103</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>Train</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonTest">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>570</y>
      <width>103</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>Test</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxTreeSamples">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>270</y>
      <width>201</width>
      <height>42</height>
     </rect>
    </property>
    <property name="minimum">
     <number>2</number>
    </property>
    <property name="maximum">
     <number>50</number>
    </property>
    <property name="singleStep">
     <number>2</number>
    </property>
    <property name="value">
     <number>3</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>131</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Tree samples</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonRegen">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>220</y>
      <width>103</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>Regen</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
